proto/umee/leverage/v1/leverage.proto

```diff
@@ -249,3 +249,43 @@ message Token {
     (gogoproto.moretags) = "yaml:\"historic_medians\""
   ];
 }
+
+// SpecialAssetPair defines a special (increased) CollateralWeight used when a specified Collateral is used
+// to collateralize a specified Borrow. This association is one-way (so it does not work in reverse).
+message SpecialAssetPair {
+  option (gogoproto.equal) = true;
+
+  // Collateral base token denom.
+  string collateral = 1;
+
+  // Borrow base token denom.
+  string borrow = 2;
+
+  // Collateral Weight defines what portion of the total value of the asset
+  // can contribute to a users borrowing power. For special asset pairs, this
+  // also overrides the borrowed asset's collateral weight when evaluating borrow
+  // factor.
+  string collateral_weight = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false,
+    (gogoproto.moretags)   = "yaml:\"collateral_weight\""
+  ];
+}
+
+// SpecialAssetSet defines a special (increased) CollateralWeight used when any of a set
+// of assets are used to borrow each other (except for looping). It is used in gov proposals
+// to create all the pairs that make up a set at once.
+message SpecialAssetSet {
+  option (gogoproto.equal) = true;
+
+  // Collateral or borrowed base token denoms.
+  repeated string assets = 1;
+
+  // Collateral Weight defines what portion of the total value of the assets
+  // can contribute to a users borrowing power, when borrowing within the set.
```

Comment:
Let's comment the valid values for the `collateral_weight`. As far as I can understand they should be [-1, 0-1], right? Let's also clarify that -1 means deletion

Reply:
I switched is to 0 for deletion and added a comment with range [0,1] for now